frontend/src/context/DataContext.tsx

```diff
@@ -132,10 +146,12 @@ export const DataContextProvider = ({ clockWorker, children }: Props) => {
                 }
               : undefined),
           };
-          const currentLap = laps[laps.length - 1];
           return {
             ...currentData,
-            graphData: [...currentData.graphData, dataPoint],
+            graphData: [
+              ...currentData.graphData,
+              { timeStamp: dataPoint.timeStamp },
+            ],
```

Comment:
Could probably be renamed to `untrackedData` - the previous idea was to display only the `graphData`, but we would not be able to have two separate colors by using a single array